### PR TITLE
Most recent Phpstorm version natively file links

### PR DESCRIPTION
**Pull request by ericroge**

### CODE REVIEW

**Comment by xabbuh** [comment]
@ericroge I wish you a happy new year. :) Can you update the pull request in the way @stof suggested?

**Comment by ericroge** [comment]
Happy new year for you also !
I've added a mention about the Mac version of PhpStorm.

**Comment by xabbuh** [comment]
Does PhpStorm handle the `pstorm` protocol only on the Mac platform?

**Comment by ericroge** [comment]
According to @ByScripts , yes.
I don't have windows, so I coan't try.

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -97,8 +97,7 @@ using the following keys:
     The ``emacs`` and ``sublime`` editors were introduced in Symfony 2.3.14.
 
 You can also specify a custom url string. If you do this, all percentage
-signs (``%``) must be doubled to escape that character. For example, if you
-have installed `PhpStormOpener`_ and use PHPstorm, you will do something like:
+signs (``%``) must be doubled to escape that character. For example, if you use PHPstorm on the Mac OS platform, you will do something like:
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: Can you please wrap lines after the first word that crosses the 72nd character?

Comment (xabbuh) [inline reference/configuration/framework.rst]: And is PhpStormOpener not needed anymore?

Comment (stof) [inline reference/configuration/framework.rst]: PhpStormOpener is Mac-only, so the new builtin feature makes it obsolete.

**Comment by stof** [comment]
PhpStormOpener is Mac-only, so it is indeed not needed anymore for PhpStorm 8. However, it might be worth linking to https://github.com/aik099/PhpStormProtocol which does not only support Mac but also Windows.

**Comment by WouterJ** [comment]
I've moved this commit into my config rewrite PR: https://github.com/WouterJ/symfony-docs/commit/d9b1a085589593d564e651aa11c6008f92f285c0 And also added a tip for windows devs: https://github.com/symfony/symfony-docs/commit/90c9ac0cae1b2036dc3078e092c77fd0eb76d642

**Comment by WouterJ** [comment]
Btw, thank you for the docs up to date with the latest changes in the dev world!

**Pull request closed without merging**